Tests/Unit/Client/ConnectionTest.php

```diff
@@ -156,10 +156,34 @@ public function testGetClient()
     public function testSetBulkParams()
     {
         $client = $this->getClient(['flush']);
-        $client->expects($this->once())->method('bulk')->with(['refresh' => 'true']);
+        $result = [
+            'refresh' => 'true',
+            'body' => [
+                0 => [
+                    'create' => [
+                        '_index' => 'foo',
+                        '_type' => 'type',
+                    ],
+                ],
+                1 => [],
+            ],
+        ];
+        $client->expects($this->once())->method('bulk')->with($result);
```

Comment:
This is nothing related to empty bulk exception functionality. Please do it as a separate PR. Remove these lines with rebase to keep history clean.